controller/rest/auth.go

```diff
@@ -2094,7 +2100,7 @@ func fedMasterTokenAuth(userName, masterToken, secret string) (*share.CLUSUser,
 		return nil, nil, err
 	}
 
-	if time.Now().Unix() >= claims.ExpiresAt {
+	if claims.ExpiresAt != nil && time.Now().After(claims.ExpiresAt.Time) {
```

Comment:
We should reject the claims without ExpiresAt, otherwise the JWT token would never expire.

Reply:
fixed